documentation/C++style.md

```diff
@@ -296,6 +297,23 @@ like `Http`.
 Consistency is one of many aspects in the pursuit of clarity,
 but not an end in itself.
 
+## <a name="checklist"></a>Pull request checklist
+Before submitting a pull request, review the following items:
+*  Verify that all files have a license.
+*  Make sure that all source lines have 80 or fewer characters.
```

Comment:
Does clang-format not cover this?

Reply:
clang-format does not handle the case where a string causes a source line to exceed 80 characters.  To handle that, you must break up the string yourself.  I will add this information.